addons/survey/models/survey_user_input.py

```diff
@@ -591,10 +633,35 @@ class SurveyUserInputLine(models.Model):
     value_text_box = fields.Text('Free Text answer')
     suggested_answer_id = fields.Many2one('survey.question.answer', string="Suggested answer")
     matrix_row_id = fields.Many2one('survey.question.answer', string="Row answer")
+    answer_display_name = fields.Char('Answer', compute="_compute_answer_display_name",
+        help="Helper field that shows the user's answer as readable text for any type of question")
     # scoring
     answer_score = fields.Float('Score')
     answer_is_correct = fields.Boolean('Correct')
 
+    @api.depends('answer_type')
```

Comment:
Not really correct (about all fields are triggers) but hey, that's just for a display_name ... as you prefer, but maybe we should put other fields quand même ?

Reply:
You did not answer yes or no or go fuck yourself. Sad tde.

Reply:
Oops forgot about that one.

Well as discussed this would imply adding *a lot* of fields there.
Probably a bit much, let's keep it like this for now.